.ci/cico-happy-path-pr-test.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# Copyright (c) 2012-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+function getOpenshiftLogs() {
+  mkdir -p /root/payload/report/oc-logs
+  cd /root/payload/report/oc-logs
+  for POD in $(oc get pods -o name); do
+    for CONTAINER in $(oc get ${POD} -o jsonpath="{.spec.containers[*].name}"); do
+      oc logs ${POD} -c ${CONTAINER} |tee $(echo ${POD}-${CONTAINER}.log | sed 's|pod/||g')
+    done
+  done
+  echo "======== oc get events ========"
+  oc get events | tee get_events.log
+  echo "======== oc get all ========"
+  oc get all | tee get_all.log
+  oc get checluster -o yaml | tee get_checluster.log || true
+}
+
+createTestWorkspaceAndRunTest() {
+  CHE_URL=$(oc get checluster eclipse-che -o jsonpath='{.status.cheURL}')
+  ### Create workspace
+  echo "====== Create test workspace ======"
+  chectl workspace:create --start --access-token "$USER_ACCESS_TOKEN" --devfile=https://raw.githubusercontent.com/eclipse/che/master/tests/e2e/files/happy-path/happy-path-workspace.yaml
```

Comment:
Typo:
-  If you put the access token in ``CHE_ACCESS_TOKEN`` environment  you don't need to use anymore --access-token flag.